src/uucore/src/lib/macros.rs

```diff
@@ -12,6 +46,45 @@ pub static UTILITY_IS_SECOND_ARG: AtomicBool = AtomicBool::new(false);
 
 //====
 
+/// Display a [`crate::error::UError`] and set global exit code.
+///
+/// Prints the error message contained in an [`crate::error::UError`] to stderr
+/// and sets the exit code through [`crate::error::set_exit_code`]. The printed
+/// error message is prepended with the calling utilities name. A call to this
```

Comment:
```suggestion
/// error message is prepended with the calling utility's name. A call to this
```

The possessive form of `utility` is `utility's`. Could you also fix this for the other macros' documentation?

Reply:
Thanks for pointing it out! I've changed it now.